src/main/java/net/openhft/chronicle/network/cluster/handlers/HeartbeatHandler.java

```diff
@@ -145,10 +145,6 @@ public void close() {
                 }
             }
             lastTimeMessageReceived = Long.MAX_VALUE;
-            Closeable closable = closable();
-            if (closable != null && !closable.isClosed()) {
-                Closeable.closeQuietly(closable);
-            }
```

Comment:
Don't close the UberHandler here, it will be closed by the TcpEventHandler when it closes